app/components/article-content.js

```diff
@@ -40,59 +40,49 @@ export default Component.extend(
 		dir: reads('wikiVariables.language.contentDir'),
 		isFastBoot: reads('fastboot.isFastBoot'),
```

Comment:
is this used somewhere? Asking because you deleted the usage of this field in .hbs